xAPI.md

```diff
@@ -414,7 +415,7 @@ examples of activities include a book, an e-learning course, a hike or a meeting
 __Activity Provider (AP)__: The software object that is communicating with 
 the LRS to record information about a learning experience. May be similar to a SCORM 
 package in that it is possible to bundle learning assets with the software object that performs this 
-communication, but an Activity Provider could also be separate from the experience it is reporting about.
+communication, but an Activity Provider could also be separate from the experience it is reporting about. This term is deprecated and will be replaced by ["Statement Provider"](#def_statement_provider) in future editions of the xAPI.
```

Comment:
I think we've learnt from experience that changing names after a lot of people are already using an existing name leads to many woes. I'm not in favor of renaming things.